spec/requests/follows_create_spec.rb

```diff
@@ -0,0 +1,41 @@
+require "rails_helper"
+
+RSpec.describe "Follows #create", type: :request do
+  let(:user) { create(:user) }
+  let(:user_2) { create(:user) }
+  let(:headers) { { "Content-Type": "application/json", Accept: "application/json" } }
+  let(:follow_payload) do
+    {
+      followable_type: "User",
+      followable_id: user_2.id,
+      verb: "follow"
+    }.to_json
+  end
+
+  before do
+    sign_in user
+  end
+
+  context "when rate limit has been hit" do
```

Comment:
Any specific reason why you choose not to test the happy path here even though you did test it in the previous spec?

Reply:
The other spec covered the true/false part of the rate limiting, there's no specific rate-limiting functionality for the case where the rate limit hasn't been hit... so I didn't think it makes sense to test the old code in a PR for a new feature.